certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/certificate/CertificateInternalApiController.java

```diff
@@ -14,10 +16,18 @@
 public class CertificateInternalApiController {
 
   private final GetCertificateInternalXmlService getCertificateInternalXmlService;
+  private final GetCertificateInternalPdfService getCertificateInternalPdfService;
 
   @PostMapping("/{certificateId}/xml")
   GetCertificateInternalXmlResponse getCertificateXml(
       @PathVariable("certificateId") String certificateId) {
     return getCertificateInternalXmlService.get(certificateId);
   }
+
+  @PostMapping("/{certificateId}/pdf")
+  GetCertificateInternalPdfResponse getCertificatePdf(
```

Comment:
We should not expose pdf-printing ni the internal api. Should be removed together with any "internal"-implementation classes